confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -301,17 +300,24 @@ def get_compatibility(self, subject=None):
         Get the current compatibility level for a subject.  Result will be one of:
 
         @:param: subject: subject name
+        @:raises: ClientError: if the request was unsuccessful or an invalid compatibility level was returned
```

Comment:
these doc strings are fixed in adminapi, make sure the merge picks it up correctly.